clang/include/clang/3C/DeclRewriter.h

```diff
@@ -105,9 +105,11 @@ class FunctionDeclBuilder : public RecursiveASTVisitor<FunctionDeclBuilder> {
 
   // Get existing itype string from constraint variables.
   std::string getExistingIType(ConstraintVariable *DeclC);
-  virtual void buildDeclVar(PVConstraint *Defn, DeclaratorDecl *Decl,
-                            std::string &Type, std::string &IType,
-                            bool &RewriteParm, bool &RewriteRet);
+
+  virtual void
+  buildDeclVar(PVConstraint *IntCV, PVConstraint *ExtCV, DeclaratorDecl *Decl,
```

Comment:
Here again I'm of a mixed mind about whether to pass the `struct` that pairs up these variables. One fewer (conceptual) parameter?